mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -116,6 +116,12 @@
     <string name="recent_tabs_header">Jump back in</string>
     <!-- Button text for showing all the tabs in the tabs tray -->
     <string name="recent_tabs_show_all">Show all</string>
+    <!-- Content description for the button which navigates the user to show all recent tabs in the tabs tray. -->
+    <string name="recent_tabs_show_all_content_description">Show all recent tabs button</string>
+    <!-- Title for showing a group item in the 'Jump back in' section of the new tab -->
```

Comment:
@rocketsroger Please always explain in comments what the placholder will be replaced with

Reply:
Sure, I'll update.  Thanks